File: Java/1008.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main{
	public static void main (String[] args){
		int a,b;
		double d;
		Scanner entrada = new Scanner (System.in);
		a = entrada.nextInt();
		b = entrada.nextInt();
		d = entrada.nextDouble();
		System.out.printf("NUMBER = %d\n", a);
		System.out.printf("SALARY = U$ %.2f\n", b*d);
	}
}

File: GO/1008.go
package main

import "fmt"

func main() {
	var a, b int
	var c float64
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&c)
	fmt.Printf("NUMBER = %d\n", a)
	d := float64(b)
	c = d * c
	fmt.Printf("SALARY = U$ %.2f\n", c)
}
